apps/web/src/components/Form.tsx

```diff
@@ -0,0 +1,39 @@
+import React, {useEffect, useRef, useState} from 'react';
+import { Stepper, Button, FormContent }  from '@components';
+
+export const Form = () => {
```

Comment:
Form seems too confusing. As this is the landing page for Planning, we can move this file content to pages

Reply:
Changed file name to Planning since its tile is Planning.